Users/coafile.rst

```diff
@@ -39,6 +39,13 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you definitely
+don't want to lint like minified files
+and backup files ( e.g. ``*.min.js``, ``*.orig``).
+
+``default_coafile`` is present in ``/coala/coalib``,
+``/coala/tests/settings/section_manager_test_files``.
```

Comment:
I don't have coala in my root directory.

Also: it's mentioned above, that:
> The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.

Seems to conflict a bit^^

Reply:
I just need to remove the initial `/` right?

Reply:
Then it's a relative part. Question is then where it refers to^^

Reply:
Just be precise ;)

Reply:
not getting u ;)

Reply:
``default_coafile`` is present in ``coalib``,
``tests/settings/section_manager_test_files``
inside the main ``coala`` repo.

Reply:
huh what? Why do also mention a tests path from the repo? The relevant information is where it should be on your system when you are a user

Reply:
Oh yeah, my mistake.

Reply:
changes made.